### PR TITLE
Add Revision History Appendix

### LINKED ISSUE #255: Add "revision history" section to specification

### PR DESCRIPTION
**Pull request by antoninbas**
We include all the significant changes from v1.0 to v1.1. Changes are
loosely ordered by "importance", i.e. how likely they are to impact
users / client & server implementations.

Fixes #255

### CODE REVIEW

**Comment by antoninbas** [comment]
@chrispsommers I don't see any inline comments from you?

File: docs/v1/P4Runtime-Spec.mdk [hunk 1]
```diff
@@ -5656,6 +5656,33 @@ man-in-the-middle attacks between the server and client.
...
 
+## Revision History
+
+### Changes in v1.1.0
+
+* Major overhaul of master-arbitration: while the Protobuf messages did not
+  change, the state machine that the server needs to implement is significantly
+  different. Upon the master disconnection, the server no longer chooses the
+  controller with the second highest election id as the new master. Instead,
+  there will not be a new master until one of the controller advertises an
```
Comment (chrispsommers) [inline docs/v1/P4Runtime-Spec.mdk]: s/controller/controllers/

**Comment by chrispsommers** [comment]
@antoninbas Sorry my comments didn't post for some reason. I see you fixed one typo (s/highest/higher) already. See the other one above: s/controller/controllers

File: docs/v1/P4Runtime-Spec.mdk (hunk 1 above)
Comment (antoninbas) [reply docs/v1/P4Runtime-Spec.mdk]: Fixed

**Comment by antoninbas** [comment]
@chrispsommers Thanks, I think it may indeed have been because I was doing the edits as you were reviewing.

**Pull request merged**